apps/frontend/src/modules/points-of-interest/slices/actions.ts

```diff
@@ -46,4 +49,16 @@ const findPaginated = createAsyncThunk<
 	return pointsOfInterest;
 });
 
-export { create, findPaginated, getById };
+const patchPointOfInterest = createAsyncThunk<
+	APIResponse<PointsOfInterestResponseDto>,
+	PatchActionPayload,
+	AsyncThunkConfig
+>(`${poiSliceName}/patch-point-of-interest`, async (payload, { extra }) => {
+	const { pointOfInterestApi } = extra;
+	const patchResult = await pointOfInterestApi.patchPointOfInterest(payload);
+	toastNotifier.showSuccess(PointOfInterestNotification.UPDATED);
+
+	return patchResult;
+});
```

Comment:
i`m not sure if you should pass an id through the payload, you can get it throug state
```suggestion
const patchPointOfInterest = createAsyncThunk<
	APIResponse<PointsOfInterestResponseDto>,
	PointOfInterestPatchRequestDto,
	AsyncThunkConfig
>(`${poiSliceName}/patch-point-of-interest`, async (payload, { extra, getState }) => {
	const { pointOfInterestApi } = extra;
	
	const state = getState()
	
	const id = state.pointsOfInterest.data?.id as number
	
	const patchResult = await pointOfInterestApi.patchPointOfInterest({id, payload});
	toastNotifier.showSuccess(PointOfInterestNotification.UPDATED);

	return patchResult;
});
```

Reply:
looks strange

Reply:
https://github.com/user-attachments/assets/8184036d-1689-4497-942d-e0a10cb2cc45

can you fix it?)

Reply:
@vyacheslavKhoroshev This is because a description field appears at the bottom

Reply:
my perfectionism gives me no rest))